includes/virtual-machines-faq-for-disks.md

```diff
@@ -271,6 +271,10 @@ Yes
 
 Yes.
 
+**Is the boot volume encrypted by default on a managed disk?**
+
+Yes. 
+
```

Comment:
Instead of this second bullet, would you mind changing the first yes (The one under "Is Azure Storage Service Encryption..." to be:

"Yes. By default, all managed disks are encrypted, including the OS disk.”

We shouldn't really discuss things at the volume level, not here anyway. Since, at the Azure level, it's just OS disk vs data disk.

Reply:
I got the question with the boot volume from a s500 customer and they needed a statement for approval - there is ton of material how other cloud provider handle this, so i think the boot volume statement does not hurt, right?